calblueprint/siren
# Bry/doc delete

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Working on ability to delete uploaded docs from firebase.

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] have all firebase docs appear in upload screen
 - [ ] update firebase accordingly when hitting "Upload" (Images that are in the imageUris array ONLY should be in the firebase when updated. Thus, hitting the "X" button on an image should delete it from the imageUris array and when "Upload" is hit, it should be removed from the firebase)

## How can the reviewer test your code? 👩‍💻
_example: run `expo start` and make sure that image docs from the firebase appear to their respective docs & cases

<img width="416" alt="Screen Shot 2022-05-17 at 4 21 22 PM" src="https://user-images.githubusercontent.com/60284767/168927736-91e97b84-fd2e-4bff-8a93-0a4e5e3318e0.png">
<img width="419" alt="Screen Shot 2022-05-17 at 4 21 36 PM" src="https://user-images.githubusercontent.com/60284767/168927756-e1e3ac26-5cb9-41e3-8a34-2343177ce750.png">


🧜‍♀️ cc: @shannonbonet @phoebeli23
